### PR TITLE
feat: add two new icons, delivered and sent

### PR DESCRIPTION
**Pull request by joelbandi**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: .changeset/tricky-readers-accept.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@twilio-paste/icons': major
```
Comment (TheSisb) [inline .changeset/tricky-readers-accept.md:2]: should be a minor
```suggestion
'@twilio-paste/icons': minor
```

File: .changeset/tricky-readers-accept.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+adds two new icons: devlivered and sent
```
Comment (TheSisb) [inline .changeset/tricky-readers-accept.md:5]:
```suggestion
[Icons] add `DeliveredIcon` and `SentIcon`
```

**Review by TheSisb — CHANGES_REQUESTED** [review]

**Comment by TheSisb** [comment]
Hi @joelbandi, Thanks for the quick fixes. You will need to also update the jest snapshot test by running `yarn jest --u` in the `paste` folder and committing the new icons list snapshot

**Comment by joelbandi** [comment]
Thank you! On it.

**Comment by TheSisb** [comment]
Closing in favor of https://github.com/twilio-labs/paste/pull/3451 to fix test and get CI running

**Pull request closed without merging**